### ISSUE TITLE
Syntax error in Python 3.7

### ISSUE THREAD

**Issue by nawatts** [Nealelab/cloudtools#83]
In Python 3.7, `async` is a reserved keyword.

https://docs.python.org/3/whatsnew/3.7.html#summary-release-highlights

Thus, this code in `stop.py` throws a SyntaxError:
https://github.com/Nealelab/cloudtools/blob/f03bdb659b66adcbeee0ee1a7370f2e57d902f74/cloudtools/stop.py#L11

**Comment by danking** [comment]
We target python2 since `gcloud` is also pinned to python2. Are you using `cloudtools` with python3 and setting the `CLOUDSDK_PYTHON` env var?

**Comment by nawatts** [comment]
I don't have `CLOUDSDK_PYTHON` set. I'm confused why `gcloud` requiring Python 2 would limit `cloudtools` to Python 2? `python3 -c 'import subprocess; subprocess.run(["gcloud", ...])` works fine.

I understand if only Python 2 is supported, but both the README and setup.py claim Python 3 support:
* https://github.com/Nealelab/cloudtools/blob/e37f3151d46d57b8b8f4e43746a8d9ab4ec0a228/README.md#installation
* https://github.com/Nealelab/cloudtools/blob/e37f3151d46d57b8b8f4e43746a8d9ab4ec0a228/setup.py#L13

**Comment by danking** [comment]
Ah, the change to `setup.py` is a recent mistaken change. The README.md is indeed wrong.

I'm curious how the `gcloud` process spawned by `subprocess.run` knows to use python2 rather than python3. Perhaps it searches your path for an executable called `python2`?

We recently set up CI & CD for cloud tools and that's when I discovered that an image with only python3 was unable to execute cloudtools. Let me try setting up an image with both python2 and python3.

**Comment by nawatts** [comment]
> Perhaps it searches your path for an executable called python2?

Yes, that's what `gcloud` does. If `CLOUDSDK_PYTHON` is not set, then it tries `python2`, then `python2.7`, then `python`, then `python3`.

According to `gcloud topic startup`, most of `gcloud` does work with Python 3.4+, though unfortunately `gsutil` does not. Since `cluster start` uses `gsutil` to fetch some files, that makes sense that an image with _only_ Python 3 would fail CI checks.
